py/desisurvey/utils.py

```diff
@@ -141,9 +141,9 @@ def equ2gal_J2000(ra_deg, dec_deg):
     dec = np.radians(dec_deg)
 
     x = np.empty(3, dtype='f8')
-    x[0] = np.cos(ra) * np.cos(dec)
-    x[1] = np.cos(ra) * np.sin(dec)
-    x[2] = np.sin(ra)
+    x[0] = np.cos(dec) * np.cos(ra)
+    x[1] = np.cos(dec) * np.sin(ra)
+    x[2] = np.sin(dec)
 
     M = np.array([ [-0.054876, -0.873437, -0.483835],
```

Comment:
An matrix of hardcoded values makes me nervous.  Can you document how the values are calculated in a comment?  Presumably roundoff error will generally leave `y.dot(y) != 1` so you might need to renormalize `y /= np.sqrt(y.dot(y))` before taking the arcsin?